js/layout-javascript/agenda-code.js

```diff
@@ -1567,14 +1586,26 @@ DynamicList.prototype.getDateIndex = function (date) {
 };
 
 DynamicList.prototype.goToToday = function () {
-  this.goToDate(moment().format('YYYY-MM-DD'));
+  var dayUserLeft = parseInt(Fliplet.Navigate.query.dayUserLeft, 10);
+
+  if (dayUserLeft) {
+    this.goToDate(dayUserLeft, true);
+    return;
+  }
+
+  this.goToDate(moment().format('YYYY-MM-DD'), false);
 };
 
-DynamicList.prototype.goToDate = function (date) {
+DynamicList.prototype.goToDate = function (date, goToDayUserLeft) {
   if (!date) {
     return;
   }
 
+  if (goToDayUserLeft) {
```

Comment:
The variable name doesn't explain very well what its purpose is. Please explain.

Reply:
@tonytlwu I added this variable so function uses information from the navigator.query which is the id of saved day or the  default date which comes in the YYYY-MM-DD format.